packages/orbit-components/src/Breadcrumbs/index.js.flow

```diff
@@ -9,7 +9,7 @@ import type { spaceAfter } from "../common/getSpacingToken/index";
 export type Props = {|
   ...Globals,
   +children: React$Node,
-  +onGoBack?: (ev: SyntheticEvent<HTMLButtonElement>) => void | Promise<any>,
+  +onGoBack?: (ev: SyntheticEvent<HTMLElement>) => void | Promise<any>,
```

Comment:
Why is this not `HTMLAnchorElement` as well?

Reply:
Because `onGoBack` is used both on `TextLink` and `Button` there's a conflict between `HTMLButtonElement` and `HTMLAnchorElement`. Therefore I used `HTMLElement`.

Reply:
Ok!